pkg/bootstrapper/bootstrapper_test.go

```diff
@@ -551,3 +551,20 @@ func testCNIUpdateKubeletArgs(t *testing.T) {
 		assert.Contains(t, err.Error(), "nil kubelet cmd passed")
 	})
 }
+
+// TestPodManifestDirCreation tests if the pod manifest directory was created
+func TestPodManifestDirCreation(t *testing.T) {
+	podManifestDirectory := "c:\\k\\etc\\kubernetes\\manifests"
+	// Create a temp directory with wmcb prefix
+	dir, err := ioutil.TempDir("", "wmcb")
+	require.NoError(t, err, "error creating temp directory")
+	// Ignore the return error as there is not much we can do if the temporary directory is not deleted
+	defer os.RemoveAll(dir)
+	wnb := winNodeBootstrapper{
+		installDir:  dir,
```

Comment:
The install directory here does not match the install directory in `podManifestDirCreation()`. This is why your test is failing. Please fix.

Reply:
Ya, just saw that too. I'll change it. Thanks for catching it.